codespell_lib/data/dictionary.txt

```diff
@@ -6913,6 +6917,7 @@ coordonated->coordinated
 coordonates->coordinates
 coorespond->correspond
 cooresponded->corresponded
+coorespondend->correspondent
```

Comment:
This probably wants to be `coorespondent` doesn't it?

Reply:
yes and no. i added the other variant too.